src/zarr/abc/store.py

```diff
@@ -221,6 +224,13 @@ def close(self) -> None:
         self._is_open = False
         pass
 
+    async def _set_dict(self, dict: Mapping[str, Buffer]) -> None:
```

Comment:
`set_many()` (analogous to [`insert_many`](https://motor.readthedocs.io/en/stable/api-tornado/motor_collection.html#motor.motor_tornado.MotorCollection.insert_many))

Reply:
I moved away from set_dict and switched to `_set_many`